cookbook/routing/extra_information.rst

```diff
@@ -7,7 +7,7 @@ How to Pass Extra Information from a Route to a Controller
 Parameters inside the ``defaults`` collection don't necessarily have to
 match a placeholder in the route ``path``. In fact, you can use the
 ``defaults`` array to specify extra parameters that will then be accessible as
-arguments to your controller:
+arguments to your controller, and as attributes of the ``Request`` object:
```

Comment:
We change this a bit and only tell the reader that we can access the parameters in the controller:

> [...] that will then be accessible in your controller::